app/queries/get_course_option_from_codes.rb

```diff
@@ -3,6 +3,11 @@ class GetCourseOptionFromCodes
   attr_accessor :provider_code, :course_code, :recruitment_cycle_year, :study_mode, :site_code,
                 :provider, :course, :site, :course_option
 
+  validates :recruitment_cycle_year, inclusion: {
```

Comment:
I think this will work if we are changing an offer, but if we aren't and are using the same course option I think this validation won't trigger?